tests/vm_test.go

```diff
@@ -325,7 +325,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			err = tests.RetryWithMetadataIfModified(vm.ObjectMeta, func(meta v12.ObjectMeta) error {
 				vm, err = virtClient.VirtualMachine(meta.Namespace).Get(meta.Name, &v12.GetOptions{})
 				Expect(err).ToNot(HaveOccurred())
-				vm.Annotations = annotations
+				vm.Spec.Template.ObjectMeta.Annotations = annotations
```

Comment:
I think that the test below: https://github.com/kubevirt/kubevirt/pull/3110/files#diff-adcd67c39cfe9de9fb883ee42a351fb4R346 will fail now.

Reply:
Yes, it's failing now in the changes of this PR (https://github.com/kubevirt/kubevirt/pull/11948). @xpivarc Was there a reason that it wasn't removed, if you remember?